### PR TITLE
changes in container

### PR DESCRIPTION
**Pull request by naghmeh-sh**
ONLY validators use jason_encode, instead of full_js

### CODE REVIEW

File: src/Talis/Html/Form/Validator/Container.php [hunk 1]
```diff
@@ -21,6 +21,8 @@ class Container{
     
     private $validators = [];
     
+    private $headers = [];
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php:24]: was already previously pushed

File: src/Talis/Html/Form/Validator/Container.php [hunk 2]
```diff
@@ -36,6 +38,17 @@ public function add_required(\Talis\Html\Form\Element\aElement $element,string $
         return $this;
     }
     
+    /***
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php:41]: was already push in previous pull request. Did u pull my branch?

File: src/Talis/Html/Form/Validator/Container.php [hunk 3]
```diff
@@ -56,42 +69,46 @@ private function add(\Talis\Html\Form\Element\aElement $element):void{
      * Generate the JS header of the validation
      * @return \stdClass
      */
-    private function header():\stdClass{
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php:59]: same as above

File: src/Talis/Html/Form/Validator/Container.php [hunk 4]
```diff
@@ -56,42 +69,46 @@ private function add(\Talis\Html\Form\Element\aElement $element):void{
...
+                    live      : 'enabled',
+                    feedbackIcons: {
+                                    valid       : 'glyphicon glyphicon-ok',
+                                    invalid     : 'glyphicon glyphicon-remove',
+                                    validating  : 'glyphicon glyphicon-refresh'
+                    }
+                   ";
         
+        foreach($this->headers as $name => $action){
+            $header .= ",".$name.":".$action;
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php:86]: should have used 
",{$name}:{$action}" syntax - pls fix!

File: src/Talis/Html/Form/Validator/Container.php [hunk 5]
```diff
@@ -56,42 +69,46 @@ private function add(\Talis\Html\Form\Element\aElement $element):void{
...
     }
     
     /**
      * Echo JS
      */
     public function __toString(){
-        $full_js = $this->header();
-        foreach($this->validators as $name => $validator){
-            $full_js->fields->{$name} = $validator;
+        $header = $this->header();
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php:95]: The header has other parts that might need to be json encoded

**Review by itay-moav — CHANGES_REQUESTED** [review]

File: src/Talis/Html/Form/Validator/Container.php (hunk 3 above)
Comment (naghmeh-sh) [reply src/Talis/Html/Form/Validator/Container.php:59]: based on new changes, header should return string

**Comment by naghmeh-sh** [comment]
another pull request exist

**Pull request closed without merging**